test/e2e/utils/testHelpers.ts

```diff
@@ -62,6 +65,72 @@ export async function addDatabaseConnection(
     if (isEnableTrustButtonVisible) {
         await enableTrustServerCertificateButton.click();
     }
+    await waitForConnectionToShowInObjectExplorer(vsCodePage, profileName);
+}
+
+export async function addDatabaseConnectionThroughWebview(
+    vsCodePage: Page,
+    serverName: string,
+    databaseName: string,
+    authType: string,
+    userName: string,
+    password: string,
+    savePassword: string,
+    profileName: string,
+): Promise<void> {
+    if (authType !== "SQL Login") return;
+
+    console.log("Connection Profile: ", serverName, databaseName, authType, userName, profileName);
```

Comment:
Maybe we use settings to add connection? That will make it UI agnostic. @Benjin any better ideas?